doc/user_guide/events.rst

```diff
@@ -0,0 +1,150 @@
+Events
+******
+
+
+.. versionadded:: 9.0
+
+Events are a mechanism to send notifications. HyperSpy events are decentralised,
+meaning that there is not a central events dispatcher. Instead, each object
+that can emit events has an :py:attr:`events`
+attribute that is an instance of :py:class:`~.events.Events` and that contains
+instances of  :py:class:`~.events.Event` as attributes. When triggered the first
+keyword argument, `obj` constains the object that the events belongs to.
+Different events may be trigger with other keyword arguments too.
```

Comment:
...may be __triggered__ ... __by__ other keyword...

Not sure about by, but sounds better for me :)